downloads/swan-lake-release-notes/swan-lake-beta1/RELEASE_NOTE.md

```diff
@@ -1,46 +1,219 @@
+
+### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Beta1](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Beta1%22+label%3AType%2FBug+label%3ATeam%2FjBallerina).
+
+## Standard Library Updates
+
+### New Features
+
+#### `graphql` Package
+- Added declarative auth configurations
+- Added support for optional types as inputs
+- Added support for returning distinct service object unions
+- Added support for inline fragments
+- Added support for enums as input values
+
+### Improvements
+
+#### `cache` Package
+Marked the `cache:Cache` class as an isolated class
+
+#### `file` Package
+Marked the `file:Listener` class as an isolated class
+
+#### `graphql` Package
+- Improved introspection validation and execution
+- Added missing fields in the GraphQL types
+- Improved the compiler plugin to validate inputs and return types
+- Added support to use included record parameters instead of the `record` type in the listener initialization
+
+#### `http` Package
+- Improved the `http:Client` remote methods to support the contextually-expected type inference
+- Changed the configuration parameters of the listeners and clients to include the record parameters
+- Populated the error detail of `http:ClientRequestError` and `http:RemoteServerError` with response payload and header
+- Swapped the `mediaType` and `headers` arguments in the client remote methods
+
+#### `sql` Package
+- Added the SQL Array Value type support and introduced the new distinct array value types for identified SQL types
+- Marked the `sql:Client` class as an isolated class
+
+#### `websubhub` Package
+Included the auth configuration to the WebSubHub publisher client configuration
+
+### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Beta1](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aclosed+is%3Aissue+milestone%3A%22Swan+Lake+Beta1%22+label%3AType%2FBug).
+
+## Code to Cloud Updates
+
+### New Features
+Added `@cloud:Expose` annotation to support custom listeners.
+
+### Bug Fixes
+
+To view bug fixes, see the GitHub milestone for Swan Lake Beta1 of the repositories below.
+
+- [C2C](https://github.com/ballerina-platform/module-ballerina-c2c/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+milestone%3A%22Ballerina+Swan+Lake+-+Beta1%22)
+- [Docker](https://github.com/ballerina-platform/module-ballerina-docker/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+milestone%3A%22Ballerina+Swan+Lake+-+Beta1%22)
+- [AWS Lambda](https://github.com/ballerina-platform/module-ballerinax-aws.lambda/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+milestone%3A%22Ballerina+Swan+Lake+-+Beta1%22)
+- [Azure Functions](https://github.com/ballerina-platform/module-ballerinax-azure.functions/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+milestone%3A%22Ballerina+Swan+Lake+-+Beta1%22) 
```

Comment:
These seem to be empty.

Reply:
@anuruddhal @xlight05

Reply:
There are no fixes in the Docker repo. Tagged the other repo issues with milestones.